frontend/components/AddressForm/index.jsx

```diff
@@ -326,13 +336,15 @@ export class AddressForm extends Component {
             {/* Default address and submit button for new address */}
             {!this.props.address.id &&
               <Fragment>
-                {splitDefaultAddressesByTags.map(tag => (
+                {this.props.config.splitDefaultAddressesByTags.map(tag => (
```

Comment:
`splitDefaultAddressesByTags` should be renamed as suggested in the comment within the `shopgate.user.getConfig.v1` pipeline definition